cloudbees-cd/es-to-os-migration/update-es-role-permissions.sh

```diff
@@ -0,0 +1,235 @@
+#!/bin/bash
+
+### Run this script using the system user assigned to run the CloudBees Analytics server
+### If you are not the user assigned to run the CloudBees Analytics, you can use `sudo su` to impersonate that user.
+### For example: if system user is analytics ###
+#
+# sudo su analytics
+#
+### end of example ###
+
+### Specify custom variables as environmental variables if required for your installation
+# ES_DIR_CONF - conf dir which contains installed config yml files
+# ES_DIR_BINS - dir with ElasticSearch binaries
+# ES_JAVA_HOME - dir with java
+# ES_PORT -  port to communicate ES, specify new port if it was not configured by default as 9300
```

Comment:
The description of `ES_PORT` needs clarification as it will be confusing to users. The default Elasticsearch HTTP port is 9200. But this port is the transport port used for communication between nodes.